content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/common/cifproductfield/CIFProductFieldHelper.java

```diff
@@ -32,6 +33,11 @@ public void activate() throws Exception {
         } else {
             selectionId = "id";
         }
+
+        filter = getRequest().getParameter("filter");
+        if (filter == null) {
+            filter = "folderOrProduct";
```

Comment:
`folderOrProduct` is used at several places, should we introduce a constant of it?